examples/planar.html

```diff
@@ -158,7 +158,7 @@
                 view.camera.camera3D.updateMatrix();
                 view.camera.camera3D.updateMatrixWorld(true);
 
-                view.notifyChange(20, true);
+                view.notifyChange(true);
```

Comment:
The delay was a left over from a test.
Here we want a continuous animation, so we want to keep the mainLoop running permanently.